### ISSUE TITLE
dock "tutorial" is not native

### ISSUE THREAD

**Issue by hervyqa** [veusz/veusz#667]
Thank you for making such a great open source software.

I found a mismatched UI when using kde plasma on linux. The "tutorial" dock doesn't use a dark theme according to the default theme (breeze dark theme).

```
Veusz version: 3.6.2
Python version: 3.10.12 (main, Jun  6 2023, 22:43:10) [GCC 12.2.0]
Python platform: linux
Numpy version: 1.24.2
Qt version: 5.15.9
PyQt version: 5.15.9
sip version: 6.6.2
```

![Screenshot_20230728_165531](https://github.com/veusz/veusz/assets/45872139/de7f441f-22fb-49b0-bfff-d18d53278815)

![Screenshot_20230728_165604](https://github.com/veusz/veusz/assets/45872139/8c0aef01-c2bf-444f-bf76-2686d3eadc11)

Scroll is also not native, unlike the scroll on the left dock.

**Comment by jeremysanders** [comment]
Yes, changing the colour stops it looking native. I don't think much can be done about that if we want to preserve the flashing elements.
